docs/source/about/installation.rst

```diff
@@ -21,13 +21,36 @@ See :ref:`Docker <about/docker:docker>` for additional information.
 
 Linux
 -----
-Follow the instructions for your preferred package type below.
+Capture Methods
```

Comment:
I'm not sure I understand why this was added to the installation section... or why some of the CUDA comments were removed.

Reply:
The sections that follow mention NvFBC, KMS etc without actually explaining what they mean. I could not find a better place for it.

No information has been removed, only extended. "CUDA is used for NVFBC capture." => "This method requires CUDA as the frame is captured to cuda memory."

Reply:
The title for the cuda compatibility table was removed... Additionally, this (`.. Tip:: See `CUDA GPUS <https://developer.nvidia.com/cuda-gpus>`_ to cross reference Compute Capability to your GPU.`) had the [admonition](https://pradyunsg.me/furo/reference/admonitions/) removed.

That table should not be within the `Packages` section. Perhaps should be given it's own section.

I think the capture methods should go under `usage` though as it has nothing to do with installation.

You can see the compiled docs for your PR here: https://lizardbyte--888.org.readthedocs.build/projects/sunshine/en/888/about/installation.html#linux

Reply:
It's still there. As part of the section.